### PR TITLE
Feature/add json accepting view

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@miltontony review please?

**Comment by codiebeulaine** [comment]
@miltontony ready for review :)

File: freebasics/views.py [hunk 1]
```diff
@@ -1,10 +1,58 @@
...
+from rest_framework import status
+
+
+class TemplateDetail(APIView):
+    """
+    Create, Retrieve, update or delete a template instance.
+    """
+    def get_object(self, pk):
+        try:
+            return FreeBasicsTemplateData.objects.get(pk=pk)
```
Comment (miltontony) [inline freebasics/views.py]: Please use [get_object_or_404](https://docs.djangoproject.com/en/1.9/topics/http/shortcuts/#get-object-or-404)

**Comment by codiebeulaine** [comment]
@miltontony ready for review

**Comment by codiebeulaine** [comment]
@miltontony @KaitCrawford ready for review

**Comment by codiebeulaine** [comment]
:+1: from milton

**Pull request merged**